CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ set(TARGET_NAME ui)
 # remove or replace with other dependencies. Note that it should also be removed
 # from vcpkg.json to prevent needlessly installing it..
 find_package(OpenSSL REQUIRED)
+include_directories(${OPENSSL_INCLUDE_DIR})
+
+# mingw only?
+set(OPENSSL_USE_STATIC_LIBS TRUE)
```

Comment:
This is likely the only change that may influence the symbols, but from [your comment](https://github.com/duckdb/duckdb-ui/issues/89#issuecomment-2754312653) it didn't work?

Reply:
I did not try separately with/without - but setting an ENV var is not the same as setting it in cmake I found in a different project, so it may make a difference.

Reply:
I've just tried locally, it doesn't seem to change anything. Does it for you?

Reply:
I now tried with this line removed, and it works still.